src/app/ReadHandler.h

```diff
@@ -267,8 +265,10 @@ class ReadHandler : public Messaging::ExchangeDelegate
     // waiting for the min reporting interval to elapse.  If we have to send a
     // report immediately due to an urgent event being queued,
     // UnblockUrgentEventDelivery can be used to force mHoldReport to false.
-    bool mHoldReport         = false;
-    bool mDirty              = false;
+    bool mHoldReport = false;
+    // The timestamp when this read handler was marked as a dirty read handler.
+    uint64_t mDirtyTick      = 0;
+    bool mOverrideDirty      = false;
```

Comment:
The interactions between these various members need to be documented.

Reply:
Added comment